packages/react-theme-provider/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": ["@uifabric/tslint-rules"],
+  "rules": {
+    "interface-name": false,
+    "jsx-ban-props": false,
+    "jsx-no-lambda": false
```

Comment:
Why disable this?

Reply:
Removed all of these.

Reply:
(except the interface one which i fixed :)